runtime/integration-tests/src/amplitude_tests.rs

```diff
@@ -16,9 +16,11 @@ use statemine_runtime as kusama_asset_hub_runtime;
 use xcm::latest::NetworkId;
 use xcm_emulator::{decl_test_network, decl_test_parachain, decl_test_relay_chain, TestExt};
 
-const KSM_FEE_WHEN_TRANSFER_TO_PARACHAIN: polkadot_core_primitives::Balance = 32000000;
-const USDT_FEE_WHEN_TRANSFER_TO_PARACHAIN: polkadot_core_primitives::Balance = 640000000;
-const NATIVE_FEE_WHEN_TRANSFER_TO_PARACHAIN: polkadot_core_primitives::Balance = 3200000000;
+// Since in the mock definition all the assets are defined with the same fee_per_second value
+// the total fees for every asset will be the same.
```

Comment:
We might want to remove this comment again.

Reply:
Yes!